pom.xml

```diff
@@ -173,20 +169,30 @@
         <dependency>
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>jakarta-mail-api</artifactId>
-            <version>2.0.1-3</version>
             <scope>compile</scope>
         </dependency>
 
     </dependencies>
 
     <build>
         <plugins>
+            <!-- TODO depend on https://plugins.jenkins.io/okhttp-api/ instead -->
             <plugin>
                 <artifactId>maven-enforcer-plugin</artifactId>
                 <executions>
                     <execution>
                         <id>display-info</id>
-                        <phase>none</phase>
+                        <configuration>
+                            <rules>
+                                <requireUpperBoundDeps>
+                                    <excludes combine.children="append">
+                                        <exclude>com.squareup.okio:okio-jvm</exclude>
+                                        <exclude>org.jetbrains.kotlin:kotlin-stdlib-common</exclude>
+                                        <exclude>org.jetbrains.kotlin:kotlin-stdlib-jdk8</exclude>
+                                    </excludes>
+                                </requireUpperBoundDeps>
+                            </rules>
+                        </configuration>
```

Comment:
Disabling Enforcer completely suppress all errors, even new violations. This change re-enables Enforcer but suppresses only existing violations, meaning that new violations will be enforced going forward. The long-term solution, described in the preceding comment, is to switch to a library plugin instead.